backends/cxxrtl/runtime/cxxrtl/cxxrtl.h

```diff
@@ -1001,16 +1001,16 @@ struct fmt_part {
 	bool signed_; // = false;
 	bool plus; // = false;
 
-	// TIME type
+	// VLOG_TIME type
 	bool realtime; // = false;
```

Comment:
This is only weakly related to this PR but still: isn't `realtime` a confusing name for this? I know the Verilog function is called `$realtime`, but I thought this had something do to with real time until I peeked into the spec just now. Could be e.g. `nonquantized`

Reply:
> isn't `realtime` a confusing name for this? I know the Verilog function is called `$realtime`, but I thought this had something do to with real time until I peeked into the spec just now. Could be e.g. `nonquantized`

It is. I think I misunderstand what it does when I added it a few years ago when I first wrote the Fmt prototype.